.github/workflow_scripts/run_benchmark.sh

```diff
@@ -14,31 +14,31 @@ source $(dirname "$0")/env_setup.sh
 setup_benchmark_env
 
 /bin/bash CI/bench/generate_bench_config.sh $MODULE $PRESET $BENCHMARK $TIME_LIMIT $BRANCH_OR_PR_NUMBER
-agbench run $MODULE"_cloud_configs.yaml" --wait
+agbench run $MODULE"_cloud_configs.yaml" --dev-branch https://github.com/autogluon/autogluon-bench.git#master --wait
```

Comment:
What is the intention of passing the -dev-branch explicitly? What is the default

Reply:
Thanks for clarification